ambry-commons/src/main/java/com/github/ambry/commons/AmbryCache.java

```diff
@@ -165,7 +171,7 @@ public boolean deleteObject(String key) {
   /**
    * Initialize metrics for this instance of AmbryCache
    */
-  private void initializeAmbryCacheMetrics() {
+  private void initializeAmbryCacheStats() {
```

Comment:
this should be metrics, because you are initializing metrics.

Reply:
just saw this comment, will fix